src/unitxt/loaders.py

```diff
@@ -198,7 +205,9 @@ def process(self):
         if self.get_limit() is not None:
             return self.limited_load()
 
-        return MultiStream.from_iterables(dataset)
+        multi_stream = MultiStream.from_iterables(dataset)
+
+        return self.add_data_classification(multi_stream)
```

Comment:
I think that instead of having every loader add this functionality, you can add create a common process() method that would return self.add_data_classification(self.load())  and each loader would loading code will be in load() and not process().  

It wil also allow us to added load_limit support in the load() method.

Reply:
I used the name `load_data` since `load` is already a method of the `Artifact` class from which loaders indirectly inherit